Resources/translations/FOSUserBundle.da.yml

```diff
@@ -52,11 +52,11 @@ registration:
             Holdet bag siden.
 
 resetting:
-    check_email: |
-        En e-mail er blevet sendt til %email%. Den indeholder et link, du skal følge for at nulstille din adgangskode.
-        Note: Du kan først bede om en ny adgangskode om %tokenLifetime% timer.
+    check_email: '|
+        En e-mail er blevet sendt til %email%. Den indeholder et link, du skal følge for at nulstille din adgangskode.'
+    Note: 'Du kan først bede om en ny adgangskode om %tokenLifetime% timer.
```

Comment:
This cannot be correct, please check the English translation: https://github.com/FriendsOfSymfony/FOSUserBundle/blob/master/Resources/translations/FOSUserBundle.en.yml#L54